gapis/api/vulkan/vulkan_terminator.go

```diff
@@ -312,7 +312,7 @@ func cutCommandBuffer(ctx context.Context, id api.CmdID,
 	// idx[3] is the secondary command buffer index inside a vkCmdExecuteCommands
 	// idx[4] is the secondary command inside the secondary command-buffer
 	submitCopy := cb.VkQueueSubmit(a.Queue, a.SubmitCount, a.PSubmits, a.Fence, a.Result)
-	submitCopy.Extras().Add(a.Extras().All()...)
+	submitCopy.Extras().CloneFromOrPanic(a.Extras().All()...)
```

Comment:
![Why not both?](http://i1.kym-cdn.com/photos/images/original/000/538/731/0fc.gif)